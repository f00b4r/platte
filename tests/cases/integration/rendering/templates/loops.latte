{for $i=0;$i<2;$i++}
Item {$i}
{/for}

{foreach [1, 2, 3] as $i}
{first}first{/first}{$i}{sep}S{/sep}{last}last{/last}
{/foreach}