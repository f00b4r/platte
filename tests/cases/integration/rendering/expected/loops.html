Item 0
Item 1

first1S
2S
3last
